pinterest/teletraan
# Notify slack/email recipients of a env/stage on unreachable instances

Status: Testing

Problem:
---
User is not notified when instance becomes unreachable. When this happens, instance could have either been dead or not able to talk to Teletraan server for various possible reasons. If instance is active, services would continue to run but, new deploy on instance would fail. 

Fix:
---
Alert slack and email recipients when this happens.

Tests:
---
1. Verify slack alert by makin instance unreachable either by removing deployd or by terminating instance using AWS console or API.
2. Verify slack alert by makin instance unreachable either by removing deployd or by terminating instance using AWS console or API.
